FirebaseAuth/Sources/User/FIRUser.m

```diff
@@ -145,6 +147,11 @@
  */
 static NSString *const kMissingUsersErrorMessage = @"users";
 
+/** @var kMissingRecaptchaTokenErrorPrefix
+    @brief The prefix of the error message of missing recaptcha token during authenticating.
+ */
+static NSString *const kMissingRecaptchaTokenErrorPrefix = @"MISSING_RECAPTCHA_TOKEN";
```

Comment:
Can we reuse https://github.com/firebase/firebase-ios-sdk/blob/760d449145a4eb7b860c5b8bf5f199c91419e8a4/FirebaseAuth/Sources/Auth/FIRAuth.m#L171?